db/seeds.rb

```diff
@@ -63,11 +178,33 @@
                       price: 300,
                       brand: 'Ride',
                       image: File.open('app/assets/images/bidndings1.jpg')
-              },
+                    },
 
                     { name: 'Anti-kickflippers',
                       description: 'No kickflips allow with the Salomon Anti-kickflippers',
                       price: 150,
                       brand: 'Salomon',
                       image: File.open('app/assets/images/bindings2.jpg')
+                    },
+                    { name: 'EX BINDINGS',
+                      description: "Built for versatility and durability, the EX snowboard binding features RIDE's all new aluminum Edge chassis to provide
+                                    all-mountain performance and flex, and Wedgie footbeds for a boost of pop and support. Topped off with the ThinGrip Plus
+                                    toe strap, RAM ankle strap and the AstroGlyde OG ratchets, you won’t find a better value for any level of rider.",
+                      price: 200,
+                      brand: 'Ride',
+                      image: File.open('app/assets/images/ex_orange.png')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.